corona-warn-app/cwa-app-android
# Contact Journal AllOver-10Minute Change (EXPOSUREAPP-9236)

This PR addresses Exposureapp-9236.
Change to 10 minute intervalls on every relevant entry inside the Journal.

HowToTest:
- Open the Contact Journal
- Create some testentries
- check the duration picker
- check the export function

Screenshots:

![1-10](https://user-images.githubusercontent.com/46747780/132848808-25be005a-05db-476b-a2f3-4146891eeb10.png)

![2-10](https://user-images.githubusercontent.com/46747780/132848811-98a42dc7-46b1-45e0-bde6-953a6ea83a83.png)

![3-10](https://user-images.githubusercontent.com/46747780/132848812-d06f5f23-1d60-4fb8-838b-d278e9e72109.png)

![image (4)](https://user-images.githubusercontent.com/46747780/132871772-61316b90-7106-4563-9c64-6507a95cfc44.png)
